backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ToplevelPhases.kt

```diff
@@ -326,7 +325,7 @@ internal val allLoweringsPhase = namedIrModulePhase(
                                 enumClassPhase then
                                 delegationPhase then
                                 callableReferencePhase then
-                                interopPart2Phase then
+                                interopPhase then
                                 varargPhase then
```

Comment:
Shalln't interopPhase be added to prereqs of varargPhase?

Reply:
Done.